.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I want an explicit reference to `ubuntu-24.04` rather than the floating reference.  We may well find that `ubuntu-26.04` needs some migration and I'd rather control that timing.

Reply:
Fair enough, done!